core/src/main/java/io/grpc/internal/GrpcAttributes.java

```diff
@@ -50,6 +50,13 @@
   public static final Attributes.Key<Boolean> ATTR_LB_PROVIDED_BACKEND =
       Attributes.Key.create("io.grpc.grpclb.lbProvidedBackend");
 
+  /**
+   * The authority to be used when constructing {@link io.grpc.Channel}s to this this EquivalentAddressGroup.
+   */
+  @EquivalentAddressGroup.Attr
+  public static final Attributes.Key<String> ATTR_CHANNEL_AUTHORITY =
+      Attributes.Key.create("io.grpc.grpclb.channelAuthority");
```

Comment:
Please rename the key string accordingly.

Reply:
@zhangkun83. Done, although I am not entirely sure about the convention here.